### PR TITLE
Release stable 11.5.1 

### PR DESCRIPTION
**Pull request by krzychu124**
Version for Stable 11.5.1 release
It's WS Labs 11.5+

[Compiled version](https://ci.appveyor.com/api/projects/krzychu124/tmpe/artifacts/TMPE.zip?branch=11.5-plus-cherry-picks)

Most of new commits were cherry-picked without changes with three exceptions:
- Apply Shift+Click speed limit to entire roundabout (copy&pasted to not peek commits from Harmony 2.x refactor)
- Bugfix floating vehicles if `ParkingAI` disabled (same as above)
- Right click (same as above)

What was skipped:
All code refactor changes, analyzers fixes,  Harmony 2.x(Cities Harmony) migration. Asset/Map editor support, MoveIt integration, `ProcessXYZ` allocations from GameBridge.dll... 
Basically everything which needs to be tested by wider user base.

### CODE REVIEW

**Review by chameleon-tbn — APPROVED** [review]
Tested ingame with 120k vanilla city with a lot of traffic:

- [x] Shift+click roundabout speed works
- [x] right click to exit UI works
- [x] no floating cars with ParkingAI disabled

all lgtm 👍

**Comment by krzychu124** [comment]
I didn't spot any issue, I'm uploading it to workshop.
